staging/src/k8s.io/kube-gen/cmd/client-gen/util/tags_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestParseTags(t *testing.T) {
+	testCases := map[string]struct {
+		lines       []string
+		expectTags  Tags
+		expectError bool
+	}{
+		"genclient": {
+			lines:      []string{`+genclient`},
```

Comment:
@sttts no, it is supported (just checking the existance of key) would you expect an error?